src/components/XBlockVideoViewer/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './XBlockVideoViewer.scss';
+
+const XBlockVideoViewer = props => (
+  <span id={props.id}>
```

Comment:
we should display the title of the video (like we're doing for image and pdf), looks like that is missing